extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/resources/dev-templates/endpoints.html

```diff
@@ -1,8 +1,8 @@
 {#include main}
 {#title}Quarkus REST Endpoints{/title}
 {#body}
-<header>
- <h4>REST Resources</h4>
+<br>
```

Comment:
In general `<br>` usage is not recommended in HTML. If you feel the margin above the `h5` is not enough you should add some CSS to this page to increase the top margin.

Reply:
Ok , I will do that :)